src/credentials/providers/ec2CredentialsProvider.ts

```diff
@@ -1,27 +1,46 @@
 /*!
- * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as AWS from 'aws-sdk'
-import { CredentialType } from '../../shared/telemetry/telemetry.gen'
-import { getStringHash } from '../../shared/utilities/textUtilities'
-import { CredentialsProvider, CredentialsProviderType ,CredentialsId } from './credentials'
-
+import { Credentials, EC2MetadataCredentials, MetadataService } from "aws-sdk"
+import { getLogger } from '../../shared/logger'
+import { CredentialType } from "../../shared/telemetry/telemetry.gen"
+import { CredentialsId, CredentialsProviderType } from './credentials'
+import { EnvironmentCredentialsProvider } from "./environmentCredentialsProvider"
```

Comment:
should be single-quotes. is prettier/husky working on your machine?

Reply:
Re-run `npm i` maybe. You'll know if Husky is active if a shell script appears somewhere in `.husky`